### DISCUSSION TITLE
Minimal stdio to HTTP SSE gateway for Claude Desktop App with DuckDB db query tool MCP server

### DISCUSSION THREAD

**Discussion by dforsber** [modelcontextprotocol/.github#45] [Show and tell]
### Pre-submission Checklist

- [X] This post follows the MCP community guidelines

## Minimal STDIO to HTTP SSE gateway

I made a minimal stdio to HTTP SSE bridge/gw with an example MCP server and client. The MCP server exposes DuckDB database and allows Claude Desktop Application to run queries on it.

### Relevant Links

https://github.com/boilingdata/mcp-server-and-gw

**Comment by PWereh** [comment]
any guide to implementation would be highly appreciated

**Comment by dforsber** [comment]
Maybe this helps? It's from my app.

![boiling-insights-doc-snippet](https://github.com/user-attachments/assets/fcde62a2-20b3-4ad1-8140-48cd04d65aa5)
